src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -394,12 +440,20 @@ void RNN::fillCellDesc() {
         inCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(shapeS, memory::data_type::f32, memory::format_tag::nc));
         outCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(shapeS, memory::data_type::f32, memory::format_tag::nc));
     }
-    if (one_of(cell_type, dnnl::algorithm::vanilla_rnn, dnnl::algorithm::vanilla_gru, dnnl::algorithm::lbr_gru, dnnl::algorithm::vanilla_lstm)) {
+
+    if (one_of(cell_type, dnnl::algorithm::vanilla_rnn, dnnl::algorithm::vanilla_gru, dnnl::algorithm::lbr_gru,
```

Comment:
Why do we need this check in general?
There are the only types we support. For example, fillSequnceDesc does not have this check.

Reply:
Sounds no need to me. @nshchego Do you happen to know why the check here historically?

Reply:
Looks like it is legacy of migration process on ngraph and can be removed.

Reply:
updated.